$OpenBSD: patch-Modules_FindwxWidgets_cmake,v 1.2 2012/05/15 08:36:28 dcoppa Exp $

From 16ee19731e13abcd7d36d18822f61b7679322631 Mon Sep 17 00:00:00 2001
From: Anthony J. Bentley <anthonyjbentley@gmail.com>
Date: Mon, 14 May 2012 15:42:00 -0400
Subject: [PATCH] FindwxWidgets: Do not use -isystem on OpenBSD (#13219)

Set wxWidgets_INCLUDE_DIRS_NO_SYSTEM on OpenBSD in addition to Apple to
prevent UsewxWidgets.cmake from using the include_directories SYSTEM
option.

--- Modules/FindwxWidgets.cmake.orig	Wed Apr 18 20:10:54 2012
+++ Modules/FindwxWidgets.cmake	Tue May 15 10:27:15 2012
@@ -167,9 +167,9 @@ SET(wxWidgets_CXX_FLAGS    "")
 #   http://www.cmake.org/pipermail/cmake/2008-April/021115.html
 #   http://www.cmake.org/pipermail/cmake/2008-April/021146.html
 #
-IF(APPLE)
+IF(APPLE OR CMAKE_CXX_PLATFORM_ID MATCHES "OpenBSD")
   SET(wxWidgets_INCLUDE_DIRS_NO_SYSTEM 1)
-ENDIF(APPLE)
+ENDIF()
 
 # DEPRECATED: This is a patch to support the DEPRECATED use of
 # wxWidgets_USE_LIBS.
